tests/test_new_command.py

```diff
@@ -47,6 +67,7 @@
         'SECRET_KEY="{{ secret_key }}"',
         "ALLOWED_HOSTS=",
         "DATABASE_URL=sqlite:///{{ project_dir }}/db.sqlite3",
+        "{% 5.1 %}" + SQLITE_OPTIONS_ENV,
```

Comment:
`{% 5.1 %}` feels like a hack, so please let me know if you have a better idea for running asserts based on the Django version.